src/telemetry/step_stats.rs

```diff
@@ -38,18 +40,19 @@ impl CsvExporter for Metrics {
         if self.print_header {
             writeln!(
                 w,
-                "Step,Records sent,Bytes sent,Indexed PRSS,Sequential PRSS"
+                "Step,Records sent,Bytes sent,Indexed PRSS,Sequential PRSS,Step narrowed"
```

Comment:
depending on whether `step-trace` feature is enabled or no, the value in this column will be either always 0, or I assume in the range [0, 1]. Either way it is not very interesting to look at I think

Reply:
Actually, this column will have more than just [0, <total_num_rows>] because some protocols skip 1 or 2 rows for optimization. For example, `BitOpStep` in `aggregate_credit::check_times_credit` sends 27 records and narrows 27 times. On the other hand, `compute_stop_bits` sends 27 records but narrows 3 times. (factor of 3 because of 3 helpers). We'll also have numbers like 0,0,0,0,3 which would tell us that the step is "no-comms". I find these quite useful.